modelservice/proxy.py

```diff
@@ -26,8 +43,62 @@ def submit_job():
     request_data: str = request.json
     print(f"request_data: {request_data}")
 
-    # TODO Parse query from HTTP JSON -> build gRPC request to Master
+    # Try to cast request data to proper types and return parameter usage error if any are incorrect
+    try:
```

Comment:
You can just use a gRPC `json_format` parser instead of doing this by hand. Check out this:
https://googleapis.dev/python/protobuf/latest/google/protobuf/json_format.html#google.protobuf.json_format.Parse

Reply:
Sweet. Updated.